backend/get_past_matches.js

```diff
@@ -0,0 +1,16 @@
+const sdk = require('api')('@developers-pandascore/v2#apvn9q1ml4sw7gei');
```

Comment:
can you explain this line, I didn't understand the parameter `@developers-pandascore/v2#apvn9q1ml4sw7gei`, Also can you write a few lines about the backed in README.md ?

Reply:
The API is provided by [Pandascore.co](https://developers.pandascore.co/reference/get_matches) as sample data, and they use the following package `@developers-pandascore/v2`

In their API reference, the # tag is persistent, as removing it results in a wrong OpenAPI Definition :
`Error: Sorry, we were unable to load that OpenAPI definition. Please either supply a URL or a path on your filesystem.`